sdk_container/src/third_party/coreos-overlay/coreos-base/coreos-cloudinit/coreos-cloudinit-9999.ebuild

```
@@ -10,7 +10,7 @@ inherit git-r3 systemd toolchain-funcs udev coreos-go
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm64"
 else
-	EGIT_COMMIT="f3aaab923de5075524780716635f25564b5e6934" # flatcar-master
+	EGIT_COMMIT="e96eae4a5b6a5967990e3e48c0197fb501d72462" # flatcar-master
```

I see this excludes the PR https://github.com/flatcar/coreos-cloudinit/pull/32
```suggestion
	EGIT_COMMIT="1c1d7f4ae6b933350d7fd36e882dda170123cccc" # flatcar-master
```